evaldo/builtins_spreadsheet.go

```diff
@@ -285,20 +292,19 @@ var Builtins_spreadsheet = map[string]*env.Builtin{
 			case env.Spreadsheet:
 				switch col := arg1.(type) {
 				case env.Block:
+					colWords := make([]env.Word, col.Series.Len())
 					for c := range col.Series.S {
 						switch ww := col.Series.S[c].(type) {
 						case env.Word:
-							res := AddIndex(ps, spr, ww)
-							switch res2 := res.(type) {
-							case env.Spreadsheet:
-								spr = res2
-							default:
-								return res
-							}
+							colWords[c] = ww
 						default:
 							return MakeError(ps, "Block of tagwords needed")
 						}
 					}
+					res := AddIndexes(ps, &spr, colWords)
+					if res != nil {
+						return res
+					}
 					return spr
```

Comment:
When adding indexes it doesn't add them in place and you have to reassign the spreadsheet (this was also the behaviour before my changes). I couldn't get it to change in place, any ideas?

Reply:
Hm, not right now. I will take note and look at it.  Otherwise not changing values in place is sort of default behavior for Rye.